components/Card.js

```diff
@@ -0,0 +1,63 @@
+import initialCards from "@/lib/db";
+import { useState } from "react";
+import styled from "styled-components";
+
+export default function Card({}) {
+  const [cards, setCards] = useState(initialCards);
+  function handleActive(id) {
+    const updatedCards = cards.map((card) => {
+      if (card.id !== id) {
+        return card;
+      } else {
+        const updatedCard = {
+          ...card,
+          isActive: !card.isActive,
+        };
+        return updatedCard;
+      }
+    });
+    setCards(updatedCards);
+    console.log(updatedCards);
+  }
+  return (
+    <>
+      <GridContainer>
+        {cards.map((card) => (
+          <GameCard
+            key={card.id}
+            type="button"
+            onClick={() => handleActive(card.id)}
+            $isActive={card.isActive}
+          >
+            {card.name} {card.icon}
+          </GameCard>
+        ))}
+      </GridContainer>
+      <p>You can add your own Playcards later on.</p>
+    </>
+  );
+}
+
+const GridContainer = styled.div`
```

Comment:
please use ul and li elements when mapping over an array (ul as container, li as children)

Reply:
Thanks, add styling in GameCrad